src/store/ducks/opensrp/assignments/index.ts

```diff
@@ -0,0 +1,159 @@
+/** Assignments redux module */
```

Comment:
Could we please add tests for this redux module?

Reply:
Test suite added!